var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -91,6 +92,7 @@ class Coreneuron(CMakePackage):
     # with the contemporary develop version it seems to work. Encode this
     # knowledge as a conflict between +report and +gpu for older versions.
     conflicts('+report', when='coreneuron@:1.0a+gpu+openmp')
+    conflicts('+report', when='coreneuron@:1.0b+gpu+openmp')
```

Comment:
I think that `:1.0b` should cover also `:1.0a`? Have you tested if this is not working for some reason maybe?

Reply:
that's right!